changelog.md

```diff
@@ -21,7 +21,12 @@
   compiler type trait instead of macro. `distinctBase` in sugar module is now deprecated.
 - `CountTable.mget` has been removed from `tables.nim`. It didn't work, and it
   was an oversight to be included in v1.0.
-
+- File handles created from high-level abstractions in the stdlib will no longer
+  be inherited by child processes. A `setInheritable` proc is also introduced
+  to allow marking explicitly which file handles should be inherited.
+  For a transistion period, define `nimInheritHandles` to enable file handle
+  inheritance by default. This flag does **not** affect the `selectors` module
+  due to the differing semantics between operating systems.
```

Comment:
But later on you write this

> File handles created via `system`, `nativesockets`, `net`, and `selectors` modules are no longer inheritable by default.

So which is it?

Reply:
This one is more of an overview, but you're right, I should clarify it.